src/test_suite.js

```diff
@@ -12,6 +12,61 @@ TestSuite.prototype.name = function (name) {
   return this;
 };
 
+TestSuite.prototype.timestamp = function (timestamp) {
+  if(_.isDate(timestamp)) timestamp = timestamp.toISOString();
+  this._attributes.timestamp = timestamp;
+  return this;
+};
+
+TestSuite.prototype.time = function (time) {
+  this._attributes.time = time;
+  return this;
+};
+
+TestSuite.prototype.tests = function (tests) {
+  this._attributes.tests = tests;
+  return this;
+};
+
+TestSuite.prototype.testsAdd = function (num) {
+  this._attributes.tests = this._attributes.tests || 0;
+  this._attributes.tests += num || 1;
+  return this;
+};
+
+TestSuite.prototype.errors = function (errors) {
+  this._attributes.errors = errors;
+  return this;
+};
+
+TestSuite.prototype.errorsAdd = function (num) {
+  this._attributes.errors  = this._attributes.errors || 0;
+  this._attributes.errors += num || 1;
+  return this;
+};
+
+TestSuite.prototype.failures = function (failures) {
+  this._attributes.failures = failures;
+  return this;
+};
+
```

Comment:
It seems so, according to [the specification](https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd#L187).